presto-jdbc/src/test/java/io/prestosql/jdbc/BaseTestJdbcResultSet.java

```diff
@@ -541,6 +547,28 @@ public void testArray()
     {
         try (ConnectedStatement connectedStatement = newStatement()) {
             checkRepresentation(connectedStatement.getStatement(), "ARRAY[1, 2]", Types.ARRAY, (rs, column) -> assertEquals(rs.getArray(column).getArray(), new int[] {1, 2}));
+
+            // array of map
```

Comment:
any guards using `getTestedPrestoServerVersion` needed?

Reply:
no, as this is client-side change only